### ISSUE TITLE
Questions about pre-trained loss and supplementary material

### ISSUE THREAD

**Issue by ReekiLee** [vt-vl-lab/DF-Net#24]
Hello!
  Thanks for your great work!
  I have a question about pre-train loss. When pre-training flow network unsupervisedly, flow network and depth-pose network are trained respectively using the same loss functions to jointly training, right? And I wonder that do all unsupervised networks need pre-training?
  By the way, I try to find the supplemetary material but I failed. Could you tell me where to find it?
  Thanks in advance!
